src/flatfiledirmgr.rs

```diff
@@ -260,6 +261,10 @@ impl<R: Runtime> NetDirProvider for FlatFileDirMgr<R> {
     fn events(&self) -> BoxStream<'static, DirEvent> {
         Box::pin(self.tx_events.subscribe())
     }
+
+    fn params(&self) -> Arc<(dyn AsRef<NetParameters> + 'static)> {
```

Comment:
```suggestion
    fn params(&self) -> Arc<dyn AsRef<NetParameters>> {
```

Reply:
Auto-complete by IntelliJ... Same thing happens in Visual Studio Code. Must be some cargo-tool.